rpcd/playbooks/roles/rpc_support/tasks/support_preinstall.yml

```diff
@@ -33,3 +33,20 @@
   when: inventory_hostname in groups['hosts']
   tags:
     - support_packages
+
+- name: Remove host packages from containers
+  apt:
+    pkg: "{{ item }}"
+    state: absent
+    update_cache: yes
+    cache_valid_time: 600
+  with_items:
+    - debbootstrap
```

Comment:
Well we even install tgtd inside the cinder api container. Our package management is not good.
Also debootstrap is not listed anywhere inside OSA.
It seems we to install lxc inside an lxc container which pulls lxc-template which pulls debootstrap so fix OSA first